code/game/objects/items/weapons/circuitboards/machinery/excelsior.dm

```diff
@@ -58,6 +58,19 @@
 	)
 	spawn_blacklisted = TRUE
 
+/obj/item/electronics/circuitboard/excelsior_autodoc
+	name = T_BOARD("excelsior autodoc")
+	build_path = /obj/machinery/excelsior_autodoc
+	board_type = "machine"
+	origin_tech = list(TECH_BIO = 5, TECH_COVERT = 2)
+	req_components = list(
+		/obj/item/stock_parts/manipulator/excelsior = 1,
+		/obj/item/stock_parts/micro_laser/excelsior = 1,
```

Comment:
Are you sure you want it to use excelsior parts exclusively?

Reply:
Yes.

Reply:
fair enough